xc7/fasm2bels/cmt_models.py

```diff
@@ -0,0 +1,164 @@
+import fasm
+from .verilog_modeling import Bel, Site
+
+# =============================================================================
+
+
+def get_pll_site(db, grid, tile, site):
+    """ Return the prjxray.tile.Site object for the given PLL site. """
+    gridinfo = grid.gridinfo_at_tilename(tile)
+    tile_type = db.get_tile_type(gridinfo.tile_type)
+
+    sites = list(tile_type.get_instance_sites(gridinfo))
+    assert len(sites) == 1, sites
+
+    return sites[0]
+
+
+def decode_multi_bit_feature(features, target_feature):
```

Comment:
I think we have a couple variants of this amoung the models.  Can we consolidate?